templates/overrides/field/field--paragraph--field-oe-title--oe-rich-text.html.twig

```diff
@@ -0,0 +1,10 @@
+{#
```

Comment:
I see it was done the same in oe_theme but I'd rather have all this paragraphs theming in one place, and crate paragraph--oe-rich-text.html.twig with
```
<h4 class="fw-bold mb-4">{{ content.field_oe_title }}</h2>
{{ content|without('field_oe_title') }}
```
I added the same class as the one in listing item block, I see in netlify in paragraphs its the same used, so lets go with this for now